blathering-barnacles/Bangazon-3
# Jb tick6 employee

## Link to Ticket
Closes #6 

## Description of Proposed Changes
Set up ability to view the list of employees and each individual employee. This ticket is not finished. Right now you are able to view the employee's first name, last name, start date, is supervisor, deleted on and url. The next PR will have the ability to view department name and computer as well as be able to put and post. I also laid out the basics of department so that I would be able to use that for employee. You should be able to see the department id, name, budget, deleted on and URL but not employees yet. That will be my next ticket. You can post and put departments.

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout jb-tick6-employee
```
Go to http://localhost:8000/api/v1/employees
You should see a list of employees and be able to click on an indiviudual URL
Go to http://localhost:8000/api/v1/employees/1
You should be able to view an employee
Repeat for departments



## Impacted Areas in Application

List general components of the application that this PR will affect:

* Employees and departments
